_includes/home/moreinfo.html

```diff
@@ -0,0 +1,12 @@
+<section id="moreinfo">
+  <div class="container">
+    <div class="col-md-7 icons-space vcenter">
+      <i class="fa fa-question-circle fa-5x" aria-hidden="true"></i>
+      <h4>Mais informações</h4>
+      <h6>Consulte o
+        <a href="/regulamento.pdf">regulamento</a> ou contacte-nos no
+        <a href="https://www.facebook.com/HackathonHeartbits/">Facebook</a>.
```

Comment:
We should change this to "Consulta" and "contacta-nos", since we are using the second person in the rest of the website